Updating title tags and meta descriptions (#2509)
pulumi/pulumi-hugo: themes/default/content/learn/abstraction-encapsulation/_index.md

---
title_tag: Abstraction & Encapsulation Overview | Learn Pulumi
title: "Abstraction and Encapsulation"
layout: module
date: 2021-11-17
description: |
    Explore abstraction and encapsulation with Pulumi and component resources.

> meta_desc: In this tutorial, we will explore how to use Pulumi to help you abstract and encapsulate your resource definitions as reusable models for others to use.

Reviewer: ```suggestion
meta_desc: In this tutorial, we will explore using Pulumi to abstract and encapsulate your resource definitions as reusable models for others to use.
```

id drop "help you" from this, as it reads kinda funny
